tapgarden/caretaker.go

```diff
@@ -60,6 +60,17 @@ type BatchCaretakerConfig struct {
 
 	GardenKit
 
+	// BroadcastCompleteChan is used to signal back to the caller that the
+	// batch has been broadcast and is now waiting for confirmation. Either
+	// this channel _or_ BroadcastErrChan is sent on, never both.
+	BroadcastCompleteChan chan struct{}
```

Comment:
I think these can instead reside on the caller level? Eventually we'll support multiple pending batches (diff namespaces) at which point these won't necessarily be global.

Reply:
Ah I see it's per caretaker, so np here imo.